app/session.go

```diff
@@ -96,7 +95,7 @@ func (a *App) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 func (a *App) UpdateSessionsIsGuest(userId string, isGuest bool) {
 	sessions, err := a.Srv.Store.Session().GetSessions(userId)
 	if err != nil {
-		mlog.Error(fmt.Sprintf("Unable to get user sessions: userId=%s err=%s", userId, err.Error()))
+		mlog.Error("Unable to get user sessions: userId=%s err=%s", mlog.String("user_id", userId), mlog.Err(err))
```

Comment:
When using structured logging, %s is not needed. This line should be: `mlog.Error("Unable to get user sessions", mlog.String("user_id", userId), mlog.Err(err))`

Reply:
Thank you for the feedback @reflog. I have made some changes. I reworded one of the error messages slightly to hopefully make more sense.